src/docs/carousel.md

```diff
@@ -0,0 +1,20 @@
+# Carousel
+
+## Content Disclaimer
+This UX pattern has been renamed from **Paging Panel**.
+
+Carousel takes users through a series of features or information pages before they start using the add-in.
+
+#### Code sample
+* [Carousel code sample](../templates/first-run/carousel)
+
+***
+
+![First Run - Carousel - Flowchart](../assets/markdown-images/carousel_flow.png)
+Recommended screen flow for when using a single identity provider in your add-in. 
+
+![First Run - Carousel - Specifications for desktop task pane](../assets/markdown-images/carousel_taskPaneCallouts.png)
+1. Carousel - Allow users to advance or skip the beginning pages of the carousel flow. 
```

Comment:
Not sure what the numbers are being used for. Can you clarify? 

We don't need the "Carousel - " at the start of 1 and 2. Can you remove please?